LambdatestEcom/Tests/APILoginTest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using LambdatestEcom.Pages;
+
+namespace LambdatestEcom.Tests
+{
+    [Parallelizable(ParallelScope.Self)]
+    [TestFixture]
+    public class APILoginTest : UITestFixture
+    {
+        [Test]
+        public async Task LoginAPI()
+        {
+            // Arrange
+            var apiLogin = new APILogin(page);
+            var userCabinet = new UserCabinetPage(page);
+
+            string email = "mymail@gmail.com";
+            string password = "test";
+
+            // Act
+            Assert.That(await apiLogin.APIlogin(email, password), Is.True); 
```

Comment:
since we can't determine if login was successful or not, there is no point to make Assert here.
Just call await apiLogin.APIlogin(email, password)
It is better to add Assert inside APIlogin or just throw exception if response is not Ok.
Assert in "Act" section is not good, moreover if it does not verify main functionality